examples/example1d_SAM_ParametricStudy/sam_template

```diff
@@ -0,0 +1,336 @@
+
```

Comment:
if this file is the same as in example1a_SAM, I recommend using symbolic link, instead of a copy.

Reply:
OK, thanks!